src/Layers/xrRender/r__dsgraph_render_lods.cpp

```diff
@@ -78,33 +79,33 @@ void R_dsgraph_structure::r_dsgraph_render_lods(bool _setup_zb, bool _clear)
 
             // gen geometry
             FLOD::_face* facets = lodV->facets;
-            svector<std::pair<float, u32>, 8> selector;
-            for (u32 s = 0; s < 8; s++)
-                selector.push_back(std::make_pair(Ldir.dotproduct(facets[s].N), s));
+            xr_vector<std::pair<float, size_t>> selector;
```

Comment:
Don't replace svector with xr_vector, because you are introducing a regression: svector is allocated in stack, while xr_vector does dynamic memory allocations in a heap, which can lead to a slowdown.